### PR TITLE
[FLEXIN-590] - pointing to correct links and documentation fix

### PR DESCRIPTION
**Pull request by AnudeepChPaul**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,11 @@ Your app will be served at http://localhost:3000/.
 
 Your application now supports query params, so that you can customise.
 1. `deploymentKey` For more info on **Deployment Key** refer to [configuration](#configuration)
-2. `region` For the host (i.e stage-us1, dev-us1, us1), defaults to us1(prod). For more info on how to find region, refer to [Configuration section](#configuration).
```
Comment (vinesh-kumar) [inline README.md:48]: why don't we need region, how do we test in lower env?

**Review by mdeshpande12 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AnudeepChPaul) [reply README.md:48]: We can still test it out, it's just that in the documentation we don't need to mention this.

Comment (vinesh-kumar) [reply README.md:48]: Do we have this information in any internal doc? like runbook?

**Review by ashishkumarTWLO — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AnudeepChPaul) [reply README.md:48]: We have this on our .env.sample file. For webchat v3, we don't have any runbook. We have `region` documented on our step by step setup internal guide.

**Review by vinesh-kumar — APPROVED** [review]

**Pull request merged**